pelutils/logger.py

```diff
@@ -2,28 +2,49 @@
 import os
 import traceback as tb
 from collections import defaultdict
+from enum import IntEnum
 from functools import update_wrapper
 from itertools import chain
 from typing import Any, Callable, DefaultDict, Generator, Iterable
 
 from pelutils import get_timestamp, get_repo
 
 
-class _Unverbose:
+class Levels(IntEnum):
+    SECTION  = 5
```

Comment:
Skal lige forstå meningen i, at section er et level (og at det er det højeste)

Reply:
Det er bare for at markere et nyt afsnit, så synes det er meget fint, at det er også nemt at se, at det er mening, at det skal være en overskrift. Desuden er det implementeret sådan, at man ikke kan lave en log uden et level